src/components/app.js

```diff
@@ -254,6 +255,23 @@ const App = () => {
                                     REORDERED_COLUMNS_PARAMETER_PREFIX_IN_DATABASE.length
                                 )
                             );
+                            const reordredParams = JSON.parse(param.value);
+                            const tabDefinition = TABLES_DEFINITION_INDEXES.get(
+                                index
+                            ).columns.map((item) => item.id);
+                            if (
+                                tabDefinition.length !== reordredParams.length
```

Comment:
```suggestion
                                tabDefinition.length > reordredParams.length
```

We only treat the "new fields" case here, not the "remove fields" one